extensions.json

```diff
@@ -180,5 +180,19 @@
     "sha256sum": "90af3124c1a68ab5b9b5d3faa480d1c77c0dd902ac1c7a71dea3f445c2df7939",
     "foxe": "https://github.com/NVIDIA-ISAAC-ROS/.github/raw/main/resources/foxglove_plugins/nvidia.nvblox_foxglove-2.1.0.foxe",
     "keywords": ["ros", "nvidia", "nvblox", "isaac"]
+  },
+  {
+    "id": "rogyken.joint-state-publisher",
+    "name": "Joint State Publisher",
+    "description": "Publishing joint state messages that can be interacted with via the slider UI.",
+    "publisher": "rogy-ken",
+    "homepage": "https://github.com/rogy-ken/foxglove-joint-state-publisher",
+    "readme": "https://github.com/rogy-ken/foxglove-joint-state-publisher/blob/main/README.md",
```

Comment:
The readme and changelog need to point to the "raw" url variants. See the Isaac ROS example above for a correct url format.

Reply:
Thank you for your review comments.

I have updated the URL as suggested. Please review the changes.